CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## [3.2.0](https://github.com/commercetools/merchant-center-application-kit/compare/v3.1.0...v3.2.0) (2018-11-30)
+
+#### 🐛 Type: Bug
+
+- `application-shell`
+  - [#162](https://github.com/commercetools/merchant-center-application-kit/pull/162) Improve permission names ([@tdeekens](https://github.com/tdeekens))
+
+#### 💥 Type: Breaking Change
```

Comment:
If it’s a Breaking change we should release `v4`

Reply:
agreed